build.gradle.kts

```diff
@@ -158,18 +158,8 @@ fun configureRepo(
     }
 
   return tasks.register("setup$taskSuffix") {
-    dependsOn(cloneTask)
     dependsOn(updateTask)
```

Comment:
Also: this fixes a race condition when executing with parallel mode (e.g. `./gradlew build --parallel`)